.eslintrc.json

```diff
@@ -72,6 +72,8 @@
         ],
         "react/no-unescaped-entities": "off",
         "react/require-default-props": "error",
+        "react/jsx-uses-react": "error",
```

Comment:
I'm not really sure what this one does. I read the docs and I'm still confused.

Reply:
I think this one I tried to attempt to resolve my error with props.children and forgot to remove it